web/docs/getting-started.md

```diff
@@ -12,8 +12,8 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 ## 1. Requirements
 
 You need to have `node` (and `npm`) installed on your machine and available in `PATH`.
-- `node`: 16.x.x
-- `npm`: 8.x.x
+- `node`: ^18.12.0
+- `npm`: ^8.19.2
```

Comment:
Yeah, I debated how to handle this one. The trick is LTS can also increment minor versions, so it won't always be `18.12.x` and could instead be `18.13.x` and above, for example. I think semver may be the best way to capture this accurately.

Here is a screenshot of the prior LTS releases:
<img width="757" alt="Screenshot 2022-10-28 123834" src="https://user-images.githubusercontent.com/523636/198688381-0bc215b5-d12b-4059-80e3-db6ede6f3d8d.png">

Reply:
Right, I completely forgot that users could (and we'd want them to) have larger minor versions too :sweat_smile:.

I guess I was left in the mindset that an LTS version starts with a `0` as its minor, which makes it a bit simpler. `x` on the minor spot basically means "anything above zero", while we can't convey "anything above 12" without a caret.

In fact, our previous specifier (`16.x.x`) was probably wrong (since the previous LTS most likely wasn't `16.0.0`), nice catch!

Anyway, all good, I'm resolving.